monkey/monkey_island/cc/services/agent_binary_service/flask_resources/agent_binaries_masque.py

```diff
@@ -0,0 +1,47 @@
+from http import HTTPStatus
+from json import JSONDecodeError
+
+from flask import make_response, request
+from flask_security import auth_token_required, roles_accepted
+
+from monkey_island.cc.flask_utils import AbstractResource
+from monkey_island.cc.services.authentication_service import AccountRole
+
+from .. import IAgentBinaryService
+
+
+class AgentBinariesMasque(AbstractResource):
+    urls = ["/api/agent-binaries/<string:os>/masque"]
+
+    def __init__(self, agent_binary_service: IAgentBinaryService):
+        self._agent_binary_service = agent_binary_service
+
+    @auth_token_required
+    @roles_accepted(AccountRole.AGENT.name, AccountRole.ISLAND_INTERFACE.name)
```

Comment:
Agents don't need this.
```suggestion
    @roles_accepted(AccountRole.ISLAND_INTERFACE.name)
```